### PR TITLE
Agoric CLI commands

### PR DESCRIPTION
**Pull request by tyg**
Documentation of Agoric CLI commands.

### CODE REVIEW

File: Agoric-CLI.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+`agoric start`
+- **Required Arguments**
+  - `[profile]`
+	- **tyg todo Not sure what goes here** ?
+  - `[args...]`
+    - **tyg todo Not sure what goes here?**	
+- **Optional Arguments**
+	-`--reset:`  Clear all VM state before starting.
+    - `--no-restart`: Do not actually start the VM. **tyg todo: Why
```
Comment (Chris-Hibbert) [inline Agoric-CLI.md]: I don't know about `agoric start` specifically, but sometimes this is useful to see complaints about the parsing of the arguments, or to place something in your command history without executing it.

File: Agoric-CLI.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+3. `agoric start` (Usually with `--reset`)
+4. `agoric deploy`
+
+`agoric init`
+- **Required Arguments**:
+    - `project`: Name of the new project to initialize.
+- **Optional Arguments**:
+    - `--dapp-template <name>`: Use the template specified by `<name>`. Defaults to the demo DApp `dapp-simple-exchange`.
+    - `--dapp-base <base-url>`: Look under this directory for the template. Defaults to `git://github.com/Agoric/`
+- **Function**: Creates a new DApp directory named `<project>` with contents copied from the `dapp-template` argument template.
```
Comment (Chris-Hibbert) [inline Agoric-CLI.md]: I'd prefer to have the **function** lines be the first thing in the description, or even to have them immediately above or below the command description without a label.

**Comment by michaelfig** [comment]
@tyg, please note that the conventional capitalisation is _Dapp_, not _DApp_.

File: Agoric-CLI.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  - None.
+- **Examples**:
+  - Installs Dapp JavaScript dependencies
+	- `agoric install`
+
+## `agoric start`
+- **Function**:
+  - Run an Agoric VM on which contracts will run.
+- **Required Arguments**:
+  - `[profile]`
```
Comment (michaelfig) [inline Agoric-CLI.md:44]: This defaults to `dev` for development mode.  You can also set it to `testnet` to connect to the current testnet.

File: Agoric-CLI.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+	- **tyg todo Not sure what goes here** ?
+  - `[args...]`
```
Comment (michaelfig) [inline Agoric-CLI.md:46]: Arguments specific to the profile.  Currently not used.

File: Agoric-CLI.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+      starting for a simulated chain to process messages.
+- **Examples**:
+  - Restart the Agoric VM, clearing all existing state before doing so.  
+    - `agoric start --reset`
+  - Don't actually start the Agoric VM.
+    - `agoric start --no-restart`
+  - For Docker-based VM, before running pull the image.
+	  - `agoric start --pull`
+    - Delay 5 seconds before starting in order for a simulated chain to
+ proces messages. (**tyg todo: Any suggestions for a good number of
```
Comment (michaelfig) [inline Agoric-CLI.md:63]: Not "before starting".  It should be "for each round-trip to the simulated chain and back".  5 is fine.  Even `1` would be fine, just to easily count the number of round trips in your head.

File: Agoric-CLI.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+You use the Agoric CLI to install dependencies, initialize, deploy, and start Agoric projects. Add one of five command names to `agoric` to specify what command to run. **Note**: Required argument names need not be given, only their values. Optional argument names must be given, along with their values. See the command's examples if you're not sure if a name is needed.
+
+In general, you want to issue Agoric CLI commands in this order:
+1. `agoric init`
+2. `agoric install`
+3. `agoric start` (Usually with `--reset`)
+4. `agoric deploy`
+
+Use `agoric help` whenever you need help with one of the above Agoric CLI commands.
+
```
Comment (michaelfig) [inline Agoric-CLI.md:12]: You should add the output from `agoric help`, which describes the global options that can be used. Notably, `-v` will produce more verbose output.

**Pull request merged**